nestkernel/ring_buffer.h

```diff
@@ -351,7 +352,30 @@ ListRingBuffer::get_index_( const delay d ) const
   assert( ( size_t ) idx < buffer_.size() );
   return idx;
 }
-}
+
+
+template < unsigned int num_values >
```

Comment:
Should we for consistency use `num_channels` instead of `num_values`?

Reply:
Yes. I have also changed the name to `MultiChannelRingBuffer` to make this more consistent.